### PR TITLE
Fix drawing labels

### PR DESCRIPTION
**Pull request by gjn**
Fix for https://github.com/geoadmin/mf-geoadmin3/issues/2691

Trivial self merge.

### CODE REVIEW

File: src/components/draw/DrawDirective.js [hunk 1]
```diff
@@ -110,6 +110,9 @@ goog.require('ga_permalink');
       var updateHelpTooltip = function(overlay, type, drawStarted, onFirstPoint,
           onLastPoint) {
         var helpMsg = $translate.instant('draw_start_' + type);
+        if (type == 'marker' || type == 'annotation') {
+          helpMsg = $translate.instant('draw_next_' + type);
```
Comment (oterral) [inline src/components/draw/DrawDirective.js]: On the screenshot of Cedric the text not translated is 'draw_start_marker' . The draw next next is not used when drawing annotation and marker.

Comment (gjn) [inline src/components/draw/DrawDirective.js]: I know, but the translation is there for next and it makes sense.

Wasn't sure if I could just replace draw_next_marker with draw_start_marker in the translations.

Comment (oterral) [inline src/components/draw/DrawDirective.js]: When you add a point there is no next message,  the help message is always "Add a point". The translation exists but it's never used so we could remove it, I guess.

Comment (oterral) [inline src/components/draw/DrawDirective.js]: SO yes replacing draw_next_marker by draw_start_marker  should work

**Pull request closed without merging**